cg/cli/workflow/microsalt/store.py

```diff
@@ -54,5 +58,16 @@ def analysis(context, config_stream):
 @store.command()
 @click.pass_context
 def completed(context):
-    """Store all completed analyses."""
-    pass
+    """Store all completed analyses"""
+    microsalt_analysis_api = context.obj["analysis_api"]
+    exit_code = EXIT_SUCCESS
+    for deliverables_file in microsalt_analysis_api.get_deliverables_to_store():
+        try:
+            context.invoke(analysis, config_stream=_io.open(deliverables_file))
+        except click.Abort:
+            exit_code = EXIT_FAIL
+        except Exception as error:
+            LOG.error(f"Unspecified error occurred - {error}")
```

Comment:
```suggestion
            LOG.error("Unspecified error occurred - %s", error)
```
[Ref](https://medium.com/swlh/why-it-matters-how-you-log-in-python-1a1085851205)